new-feature.txt

```diff
@@ -0,0 +1 @@
+This file is created in new feture1 branch
```

Comment:
It's really good that you have decided to add this line of code

Reply:
thnks for this comment